lib/main.dart

```diff
@@ -17,7 +20,22 @@ class MyApp extends StatelessWidget {
             textTheme: TextTheme(
               headline: TextStyle(fontSize: 36, fontWeight: FontWeight.bold),
               subhead: TextStyle(fontSize: 17.0, fontWeight: FontWeight.bold),
+              display1: TextStyle(fontSize: 17.0, color: Colors.black),
             )),
         home: Login());
   }
 }
+
+String localhost() {
+  if(Platform.isAndroid) {
+    return 'http://10.0.2.2:3000'; // works for emulator
+  } else {
+    return 'http://localhost:3000';
+  }
```

Comment:
Is there a way to configure this via env-var? This will be a nightware when you have local, staging, and prod server.